### PR TITLE
Update login button

### PR DESCRIPTION
**Pull request by ozermzn**
Loading design for login button has been added.
Update login button #15344

### CODE REVIEW

**Comment by rqx110** [comment]
⚠️ it looks like your commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

File: npm/ng-packs/apps/dev-app/src/app/home/home.component.ts [hunk 1]
```diff
@@ -1,19 +1,21 @@
-import { AuthService } from '@abp/ng.core';
+import { AuthService, RouterEvents } from '@abp/ng.core';
 import { Component } from '@angular/core';
-import { OAuthService } from 'angular-oauth2-oidc';
+import { Event, NavigationCancel, NavigationEnd, NavigationStart, Router } from '@angular/router';
```
Comment (rqx110) [inline npm/ng-packs/apps/dev-app/src/app/home/home.component.ts]: not used
```suggestion

```

File: npm/ng-packs/apps/dev-app/src/app/home/home.component.ts [hunk 2]
```diff
@@ -1,19 +1,21 @@
-import { AuthService } from '@abp/ng.core';
+import { AuthService, RouterEvents } from '@abp/ng.core';
```
Comment (rqx110) [inline npm/ng-packs/apps/dev-app/src/app/home/home.component.ts]: not used
```suggestion
import { AuthService } from '@abp/ng.core';
```

File: npm/ng-packs/apps/dev-app/src/app/home/home.component.ts [hunk 3]
```diff
@@ -1,19 +1,21 @@
...
 import { Component } from '@angular/core';
-import { OAuthService } from 'angular-oauth2-oidc';
+import { Event, NavigationCancel, NavigationEnd, NavigationStart, Router } from '@angular/router';
+import { map } from 'rxjs';
```
Comment (rqx110) [inline npm/ng-packs/apps/dev-app/src/app/home/home.component.ts]: not used
```suggestion

```

File: npm/ng-packs/apps/dev-app/src/app/home/home.module.ts [hunk 4]
```diff
@@ -2,9 +2,10 @@ import { NgModule } from '@angular/core';
 import { SharedModule } from '../shared/shared.module';
 import { HomeRoutingModule } from './home-routing.module';
 import { HomeComponent } from './home.component';
+import { LoadingDirective, ThemeSharedModule } from '@abp/ng.theme.shared';
```
Comment (rqx110) [inline npm/ng-packs/apps/dev-app/src/app/home/home.module.ts]: no need this,
```suggestion

```

File: npm/ng-packs/apps/dev-app/src/app/home/home.module.ts [hunk 5]
```diff
@@ -2,9 +2,10 @@ import { NgModule } from '@angular/core';
...
 
 @NgModule({
   declarations: [HomeComponent],
-  imports: [SharedModule, HomeRoutingModule],
+  imports: [SharedModule, HomeRoutingModule, ThemeSharedModule],
```
Comment (rqx110) [inline npm/ng-packs/apps/dev-app/src/app/home/home.module.ts]: `ThemeSharedModule` has been imported in `SharedModule`
```suggestion
  imports: [SharedModule, HomeRoutingModule],
```

**Review by rqx110 — CHANGES_REQUESTED** [review]

File: npm/ng-packs/apps/dev-app/src/app/home/home.module.ts (hunk 5 above)
Comment (ozermzn) [reply npm/ng-packs/apps/dev-app/src/app/home/home.module.ts]: Thanks you for your feedbacks, i will fix it.

**Review by mahmut-gundogdu — APPROVED** [review]

**Pull request merged**